### PR TITLE
ENH : Update the SOPs specifying the ET-to-BIDS conversion

### LINKED ISSUE #342: Rewrite ET-to-BIDS conversion in the SOPs
Update the following section of the SOPs:

https://github.com/TheAxonLab/hcph-sops/blob/91121e8e465e4e53427b0ad053aacc04bca58c50/docs/data-management/post-session.md#L288-L359

With the following goals:

- [ ] Clearly describe that we need to use a docker image of bidsphysio with @esavary's code. If there is a built image of this, please push it to docker hub so that we don't need to build it ourselves.
- [ ] Clearly state the command line necessary to convert the EDF file corresponding to a task.
- [ ] Include comments that are relevant regarding how to run it best.
- [ ] Indicate whatever clerical stuff (move files, rename files, complete metadata in JSONs) that is necessary.
- [ ] Add a schedule TSV file like those we have in `code/events/` and `code/physioconv/`, where session number, date, phase encoding direction and corresponding EDF files are listed (see [`code/events/schedule.tsv`](https://github.com/TheAxonLab/hcph-sops/blob/mkdocs/code/events/schedule.tsv) for the closest example of this)

### PR DESCRIPTION
**Pull request by esavary**
Update the SOPs with the following goals:

 - Clarify the need for using the bidsphysio Docker image with @esavary's code. ( I pushed a pre-built image to Docker Hub for convenience.)
 - Clearly state the command line for EDF file conversion.
 - Add all relevant comments for optimal execution.
 -  Document all metadata necessary to complete the task.
 - Includes a schedule TSV file for session and data overview.

closes #342

### CODE REVIEW

File: docs/data-management/post-session.md [hunk 1]
```diff
@@ -286,8 +286,19 @@ To support backward compatibility (and some extra, currently unsupported feature
     ```
 
 ### Convert eye-tracking into BIDS with *bidsphysio*
+- [ ] Pull the latest docker image of esavary/bidsphysio with:
+``` shell
+docker pull esavary/bidsphysio
+```
```
Comment (oesteban) [inline docs/data-management/post-session.md]:
```suggestion
    ``` shell
    docker pull esavary/bidsphysio
    ```
```

File: docs/data-management/post-session.md [hunk 2]
```diff
@@ -286,8 +286,19 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+- [ ] Open [schedule_ET_data.tsv](../code/physioconv/schedule_ET_data.tsv) to find the phase encoding direction and the name of the .EDF file corresponding to the session you want to convert.
```
Comment (oesteban) [inline docs/data-management/post-session.md]:
```suggestion
- [ ] Open [`schedule_ET_data.tsv`](../code/physioconv/schedule_ET_data.tsv) to find the phase encoding direction and the name of the `.EDF` file corresponding to the session you want to convert.
```

File: docs/data-management/post-session.md [hunk 3]
```diff
@@ -286,8 +286,19 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+- [ ] In the folder containing your data, create a new folder named metadata. Copy the file containing information about the eye-tracker, named [info_ET.json](../code/physioconv/info_ET.json), inside this metadata folder.
```
Comment (oesteban) [inline docs/data-management/post-session.md]: Please start every sentence in a new line. Beyond the stylistic comment, I cannot understand this. Could you rephrase?
```suggestion
- [ ] In the folder containing your data, create a new folder named metadata.
    Copy the file containing information about the ET, named [`info_ET.json`](../code/physioconv/info_ET.json), inside this metadata folder.
```

File: docs/data-management/post-session.md [hunk 4]
```diff
@@ -286,8 +286,19 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+- [ ]  Run the following command for the file corresponding to the DWI (Diffusion-Weighted Imaging) data, replacing PATH_RAW_DATA,PATH_OUTPUT, DWI_FILENAME, SESSION_NUMBER, and PHASE_ENCODING_DIRECTION with the information corresponding to the file you want to process:
```
Comment (oesteban) [inline docs/data-management/post-session.md]: Please use real names (examples) rather than placeholders.
```suggestion
- [ ]  Run the following command for the file corresponding to the DWI (Diffusion-Weighted Imaging) data, replacing PATH_RAW_DATA,PATH_OUTPUT, DWI_FILENAME, SESSION_NUMBER, and PHASE_ENCODING_DIRECTION with the information corresponding to the file you want to process:
```

File: docs/data-management/post-session.md [hunk 5]
```diff
@@ -286,8 +286,19 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+``` shell
+docker run -u $( id -u ):$( id -g ) --rm -it -v PATH_RAW_DATA:/data -v PATH_OUTPUT:/output --entrypoint=/opt/venv/bin/python esavary/bidsphysio /opt/venv/bin/edf2bidsphysio --infile DWI_FILENAME --bidsprefix /output/dwi/sub-001_ses-SESSION_NUMBER_acq-highres_dir-PHASE_ENCODING_DIRECTION -m /data/metadata/info_ET.json
+
+```
```
Comment (oesteban) [inline docs/data-management/post-session.md]:
```suggestion
    ``` shell
    docker run -u $( id -u ):$( id -g ) --rm -it \
        -v PATH_RAW_DATA:/data -v PATH_OUTPUT:/output \
        --entrypoint=/opt/venv/bin/python esavary/bidsphysio \
        /opt/venv/bin/edf2bidsphysio --infile DWI_FILENAME \
        --bidsprefix /output/dwi/sub-001_ses-SESSION_NUMBER_acq-highres_dir-PHASE_ENCODING_DIRECTION \
        -m /data/metadata/info_ET.json
    ```
```

File: docs/data-management/post-session.md [hunk 6]
```diff
@@ -286,8 +286,19 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+!!! In addition to the eyetracker data and metadata JSON file, the code also generates a .tsv file containing all messages sent to the eye-tracker and the header of the .EDF file (named ..._eventlist_raw.tsv). Please exercise caution and ensure that you do not copy these generated files into the BIDS-compatible repository."
```
Comment (oesteban) [inline docs/data-management/post-session.md]: This will not render properly, which makes me think that should be checked before next iterations of review.
```suggestion
    !!! danger "Do not copy all output files directly"
    
        In addition to the eye-tracking data (`<prefix>_eyetrack.tsv.gz` file) and metadata (`<prefix>_eyetrack.json` file), the code also generates a TSV file containing all messages sent to the ET and the header of the `.EDF` file (with name `<prefix>_eventlist_raw.tsv`).
        Please do not copy these generated files into the *DataLad* dataset in BIDS.
```

File: docs/data-management/post-session.md [hunk 7]
```diff
@@ -286,8 +286,19 @@ To support backward compatibility (and some extra, currently unsupported feature
...
 
-We will adapt [*bidsphysio*](https://github.com/cbinyu/bidsphysio) to convert eye-tracking from the EDF proprietary format of EyeLink (SR Instruments).
 
 !!! warning "Instead of the current specifications, we will use [the following BEP](https://bids-specification--1128.org.readthedocs.build/en/1128/modality-specific-files/eye-tracking.html)"
```
Comment (oesteban) [inline docs/data-management/post-session.md]: This needs to be updated and possibly moved before the new edits.

File: code/physioconv/info_ET.json
Comment (oesteban) [inline code/physioconv/info_ET.json:1]: Can we put this file and the schedule TSV under a new folder (and the upcoming notebook)?

e.g.,

```
code/eyetracking/info_ET.json
code/eyetracking/schedule.tsv
```

File: docs/data-management/post-session.md [hunk 8]
```diff
@@ -286,8 +286,23 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+- [ ]  Run the following command for the file corresponding to the DWI (Diffusion-Weighted Imaging) data, replacing PATH_RAW_DATA,PATH_OUTPUT, DWI_FILENAME, SESSION_NUMBER, and PHASE_ENCODING_DIRECTION with the information corresponding to the file you want to process:
+``` shell
+docker run -u $( id -u ):$( id -g ) --rm -it -v PATH_RAW_DATA:/data -v PATH_OUTPUT:/output --entrypoint=/opt/venv/bin/python esavary/bidsphysio /opt/venv/bin/edf2bidsphysio --infile DWI_FILENAME --bidsprefix /output/dwi/sub-001_ses-SESSION_NUMBER_acq-highres_dir-PHASE_ENCODING_DIRECTION -m /data/metadata/info_ET.json
+
+```
+- [ ] Run the following command for the three files corresponding to the functional tasks:
+``` shell
+docker run -u $( id -u ):$( id -g ) --rm -it -v PATH_RAW_DATA:/data -v PATH_OUTPUT:/output --entrypoint=/opt/venv/bin/python bidsphysio /opt/venv/bin/edf2bidsphysio --infile /data/FILENAME --bidsprefix /output/session01/func/sub-001_ses-SESSION_NUMBER_task-TASK_NAME_dir-PHASE_ENCODING_DIRECTION -m /data/metadata/info_ET.json
+```
```
Comment (oesteban) [inline docs/data-management/post-session.md]:
```suggestion
- [ ] Run the following command for the three files corresponding to the functional tasks:
    ``` shell
    docker run -u $( id -u ):$( id -g ) --rm -it \
        -v PATH_RAW_DATA:/data -v PATH_OUTPUT:/output \
        --entrypoint=/opt/venv/bin/python bidsphysio \
        /opt/venv/bin/edf2bidsphysio \
        --infile /data/FILENAME \
        --bidsprefix /output/session01/func/sub-001_ses-SESSION_NUMBER_task-TASK_NAME_dir-PHASE_ENCODING_DIRECTION -m /data/metadata/info_ET.json
    ```
```

**Comment by oesteban** [comment]
> * Clearly state the command line for EDF file conversion.

This is the correct spelling of conversion. AFAIK "convertion" is not an English word. I changed the title of the PR to be more descriptive of what is going on here.

Beyond the minor details, this is going in a very good direction!

File: docs/data-management/post-session.md (hunk 4 above)
Comment (esavary) [reply docs/data-management/post-session.md]: I think it can make it more confusing as to what exactly needs to be changed and where when running the command, especially when directly including the session number and phase encoding direction. Would it not be better to include this in the notebook or as an example?

File: code/physioconv/info_ET.json
Comment (esavary) [reply code/physioconv/info_ET.json:1]: Sure, I'll create a new folder

File: docs/data-management/post-session.md (hunk 4 above)
Comment (oesteban) [reply docs/data-management/post-session.md]: I don't think the notebook is a replacement for this. I'm happy also to have this example in the notebook, but I believe this is easier to understand with specific examples.

If it were only one field, then I'd probably be okay with it, but the number of parameters to replace is a mouthful and they only extend the example code (e.g., `PHASE_ENCODING_DIRECTION` instead of just `LR`).

Comment (esavary) [reply docs/data-management/post-session.md]: As you prefer. I've updated it with a real example.

Comment (oesteban) [reply docs/data-management/post-session.md]: Thx!

File: docs/data-management/post-session.md [hunk 9]
```diff
@@ -287,9 +287,36 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+docker run -u $( id -u ):$( id -g ) --rm -it \
+-v /home/esavary/Projects/HCPH-BIDS_ET/data/:/data -v /home/esavary/Projects/HCPH-BIDS_ET/output/:/output \
+--entrypoint=/opt/venv/bin/python bidsphysio /opt/venv/bin/edf2bidsphysio \
+--infile /data/qct_2023-10-20_19h40.38.964_2_session_1.EDF \
+--bidsprefix /output/session01/func/sub-001_ses-001_task-qct_dir-LR -m /data/metadata/info_ET.json
+
+    ```
+!!! danger "Do not copy all output files directly"
+    In addition to the eye-tracking data (`<prefix>_eyetrack.tsv.gz` file) and metadata (`<prefix>_eyetrack.json` file), the code also generates a TSV file containing all messages sent to the ET and the header of the `.EDF` file (with name `<prefix>_eventlist_raw.tsv`).
+        Please do not copy these generated files into the *DataLad* dataset in BIDS.
```
Comment (oesteban) [inline docs/data-management/post-session.md]: For some reason, my suggestion has been modified. Without the correct blank lines and indentation, this will not render properly (please check before requesting another round):

```suggestion

!!! danger "Do not copy all output files directly"

    In addition to the eye-tracking data (`<prefix>_eyetrack.tsv.gz` file) and metadata (`<prefix>_eyetrack.json` file), the code also generates a TSV file containing all messages sent to the ET and the header of the `.EDF` file (with name `<prefix>_eventlist_raw.tsv`).
    Please do not copy these generated files into the *DataLad* dataset in BIDS.
```

File: docs/data-management/post-session.md [hunk 10]
```diff
@@ -287,9 +287,36 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+- [ ] Run the following command for the files corresponding to the functional tasks:
+    ``` shell
+    
+docker run -u $( id -u ):$( id -g ) --rm -it \
+-v /home/esavary/Projects/HCPH-BIDS_ET/data/:/data -v /home/esavary/Projects/HCPH-BIDS_ET/output/:/output \
+--entrypoint=/opt/venv/bin/python bidsphysio /opt/venv/bin/edf2bidsphysio \
+--infile /data/qct_2023-10-20_19h40.38.964_2_session_1.EDF \
+--bidsprefix /output/session01/func/sub-001_ses-001_task-qct_dir-LR -m /data/metadata/info_ET.json
+
+    ```
```
Comment (oesteban) [inline docs/data-management/post-session.md]: Here, the spacings and indentations were removed. Whatever tool did this you should be careful not to use it or tame it:

```suggestion
    ``` shell
    docker run -u $( id -u ):$( id -g ) --rm -it \
        -v <path-to-EDF-data>:/data \
        -v <output-path-on-host>:/output \
        --entrypoint=/opt/venv/bin/python bidsphysio /opt/venv/bin/edf2bidsphysio \
        --infile /data/qct_2023-10-20_19h40.38.964_2_session_1.EDF \
        --bidsprefix /output/session01/func/sub-001_ses-001_task-qct_dir-LR \
        -m /data/metadata/info_ET.json
    ```
```

File: docs/data-management/post-session.md [hunk 11]
```diff
@@ -287,9 +287,36 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+- [ ] Create a new folder named metadata inside the folder containing your .EDF file.
+    Copy the file containing information about the ET, named [`info_ET.json`](../code/physioconv/info_ET.json), inside this metadata folder.
+- [ ]  Run the following command for the file corresponding to the DWI (Diffusion-Weighted Imaging) data, replacing PATH_RAW_DATA,PATH_OUTPUT, DWI_FILENAME, SESSION_NUMBER, and PHASE_ENCODING_DIRECTION with the information corresponding to the file you want to process:
+    ``` shell
+    docker run -u $( id -u ):$( id -g ) --rm -it \
+    -v /home/esavary/Projects/HCPH-BIDS_ET/data/:/data -v /home/esavary/Projects/HCPH-BIDS_ET/output/:/output \
+    --entrypoint=/opt/venv/bin/python bidsphysio /opt/venv/bin/edf2bidsphysio \
+    --infile /data/fixation_2023-10-20_18h48.03.561_5_session_1.EDF \
+    --bidsprefix /output/session01/dwi/sub-001_ses-001_acq-highres_dir-LR -m /data/metadata/info_ET.json
+    ```
```
Comment (oesteban) [inline docs/data-management/post-session.md]:
```suggestion
    ``` shell
    docker run -u $( id -u ):$( id -g ) --rm -it \
        -v <path-to-EDF-data>:/data \
        -v <output-path-on-host>:/output \
        --entrypoint=/opt/venv/bin/python bidsphysio /opt/venv/bin/edf2bidsphysio \
        --infile /data/fixation_2023-10-20_18h48.03.561_5_session_1.EDF \
        --bidsprefix /output/session01/dwi/sub-001_ses-001_acq-highres_dir-LR \
        -m /data/metadata/info_ET.json
    ```
```

File: docs/data-management/post-session.md [hunk 12]
```diff
@@ -287,9 +287,36 @@ To support backward compatibility (and some extra, currently unsupported feature
...
 
-!!! warning "Instead of the current specifications, we will use [the following BEP](https://bids-specification--1128.org.readthedocs.build/en/1128/modality-specific-files/eye-tracking.html)"
+- [ ] Pull the latest docker image of esavary/bidsphysio with:
+    ``` shell
+    docker pull esavary/bidsphysio
+    ```
+- [ ] Open [`schedule_ET_data.tsv`](../code/physioconv/schedule_ET_data.tsv) to find the phase encoding direction and the name of the `.EDF` file corresponding to the session you want to convert.
+- [ ] Create a new folder named metadata inside the folder containing your .EDF file.
+    Copy the file containing information about the ET, named [`info_ET.json`](../code/physioconv/info_ET.json), inside this metadata folder.
+- [ ]  Run the following command for the file corresponding to the DWI (Diffusion-Weighted Imaging) data, replacing PATH_RAW_DATA,PATH_OUTPUT, DWI_FILENAME, SESSION_NUMBER, and PHASE_ENCODING_DIRECTION with the information corresponding to the file you want to process:
```
Comment (oesteban) [inline docs/data-management/post-session.md]:
```suggestion
- [ ]  Execute the ET to BIDS conversion on the dMRI data.
    Run the following command for the corresponding file in the [`schedule.tsv`](../code/eyetracking/schedule.tsv) file:
```

File: docs/data-management/post-session.md [hunk 13]
```diff
@@ -287,9 +287,34 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+    --bidsprefix /output/session01/dwi/sub-001_ses-001_acq-highres_dir-LR -m /data/metadata/info_ET.json
+    ```
+- [ ] Run the following command for the files corresponding to the functional tasks:
+    ``` shell
+    docker run -u $( id -u ):$( id -g ) --rm -it \
+    -v /home/esavary/Projects/HCPH-BIDS_ET/data/:/data -v /home/esavary/Projects/HCPH-BIDS_ET/output/:/output \
+    --entrypoint=/opt/venv/bin/python bidsphysio /opt/venv/bin/edf2bidsphysio \
+    --infile /data/qct_2023-10-20_19h40.38.964_2_session_1.EDF \
+    --bidsprefix /output/session01/func/sub-001_ses-001_task-qct_dir-LR -m /data/metadata/info_ET.json
+    ```
```
Comment (oesteban) [inline docs/data-management/post-session.md]:
```suggestion
    ``` shell
    docker run -u $( id -u ):$( id -g ) --rm -it \
        -v <path-to-EDF-data>:/data \
        -v <output-path-on-host>:/output \
        --entrypoint=/opt/venv/bin/python bidsphysio /opt/venv/bin/edf2bidsphysio \
        --infile /data/qct_2023-10-20_19h40.38.964_2_session_1.EDF \
        --bidsprefix /output/session01/func/sub-001_ses-001_task-qct_dir-LR \
        -m /data/metadata/info_ET.json
    ```
```

**Review by oesteban — COMMENTED** [review]
The tool you are using for editing is modifying indentation and spaces and that is a bit irritating...

File: docs/data-management/post-session.md [hunk 14]
```diff
@@ -287,9 +287,34 @@ To support backward compatibility (and some extra, currently unsupported feature
...
 
-We will adapt [*bidsphysio*](https://github.com/cbinyu/bidsphysio) to convert eye-tracking from the EDF proprietary format of EyeLink (SR Instruments).
+!!! warning "Instead of the current specifications, we are using [the following BEP](https://bids-specification--1128.org.readthedocs.build/en/1128/modality-specific-files/eye-tracking.html)"
 
-!!! warning "Instead of the current specifications, we will use [the following BEP](https://bids-specification--1128.org.readthedocs.build/en/1128/modality-specific-files/eye-tracking.html)"
+- [ ] Pull the latest docker image of esavary/bidsphysio with:
+    ``` shell
+    docker pull esavary/bidsphysio
+    ```
+- [ ] Open [`schedule_ET_data.tsv`](../code/physioconv/schedule_ET_data.tsv) to find the phase encoding direction and the name of the `.EDF` file corresponding to the session you want to convert.
```
Comment (oesteban) [inline docs/data-management/post-session.md]:
```suggestion
- [ ] Open [`schedule.tsv`](../code/eyetracking/schedule.tsv) to find the phase encoding direction and the name of the `.EDF` file corresponding to the session you want to convert.
```

File: docs/data-management/post-session.md [hunk 15]
```diff
@@ -287,9 +287,34 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+- [ ] Create a new folder named metadata inside the folder containing your .EDF file.
```
Comment (oesteban) [inline docs/data-management/post-session.md]: Is this what you  mean?

```suggestion
- [ ] Create a new folder named `metadata/` inside the folder containing the target `.EDF` file.
```

File: docs/data-management/post-session.md [hunk 16]
```diff
@@ -287,9 +287,34 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+    Copy the file containing information about the ET, named [`info_ET.json`](../code/physioconv/info_ET.json), inside this metadata folder.
```
Comment (oesteban) [inline docs/data-management/post-session.md]:
```suggestion
    Copy the file containing information about the ET, named [`info_ET.json`](../code/eyetracking/info_ET.json) into the `metadata/` folder.
```

File: docs/data-management/post-session.md (hunk 10 above)
Comment (oesteban) [reply docs/data-management/post-session.md]: This one was updated while I was writing. Marking resolved.

**Review by oesteban — CHANGES_REQUESTED** [review]
I left a bunch of comments above before you requested the review again :D

**Comment by oesteban** [comment]
> * I pushed a pre-built image to Docker Hub for convenience

Checked this, and it indeed works - thanks :+1:

File: docs/data-management/post-session.md (hunk 9 above)
Comment (oesteban) [reply docs/data-management/post-session.md]: BTW, this last admonition has the implicit message that *some* undisclosed files MUST be copied (as opposed those that don't), so I'm guessing one more checkbox indicating these files should be added somewhere around here.

**Review by oesteban — COMMENTED** [review]
There's one remaining comment https://github.com/TheAxonLab/hcph-sops/pull/346/files/3b6c67d843ddcf2ea4d46141bfd3c63355ed43d2#r1386885460

Once that is clarified, this is a go!

**Comment by esavary** [comment]
> There's one remaining comment https://github.com/TheAxonLab/hcph-sops/pull/346/files/3b6c67d843ddcf2ea4d46141bfd3c63355ed43d2#r1386885460
> 
> Once that is clarified, this is a go!

Does the link lead to the right place? It seems to redirect to a comment I've replied to.

File: docs/data-management/post-session.md [hunk 17]
```diff
@@ -287,9 +287,41 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+        --entrypoint=/opt/venv/bin/python bidsphysio /opt/venv/bin/edf2bidsphysio \
+        --infile /data/qct_2023-10-20_19h40.38.964_2_session_1.EDF \
+        --bidsprefix /output/session01/func/sub-001_ses-001_task-qct_dir-LR \
+        -m /data/metadata/info_ET.json
+    ```
+
+!!! danger "Do not copy all output files directly"
+
+    In addition to the eye-tracking data (`<prefix>_eyetrack.tsv.gz` file) and metadata (`<prefix>_eyetrack.json` file), the code also generates a TSV file containing all messages sent to the ET and the header of the `.EDF` file (with name `<prefix>_eventlist_raw.tsv`).
+    Please do not copy these generated files into the *DataLad* dataset in BIDS.
```
Comment (oesteban) [inline docs/data-management/post-session.md:325]: BTW, this last admonition has the implicit message that some undisclosed files MUST be copied (as opposed those that don't), so I'm guessing one more checkbox indicating these files should be added somewhere around here.

**Review by oesteban — COMMENTED** [review]
Weirdly, I could see the comment. Anyways, I just added it again. The comment only points out that there is no rounding up saying "copy these files from here to there"

File: docs/data-management/post-session.md [hunk 18]
```diff
@@ -287,9 +287,42 @@ To support backward compatibility (and some extra, currently unsupported feature
...
+    ``` shell
+    docker run -u $( id -u ):$( id -g ) --rm -it \
+        -v <path-to-EDF-data>:/data \
+        -v <output-path-on-host>:/output \
+        --entrypoint=/opt/venv/bin/python bidsphysio /opt/venv/bin/edf2bidsphysio \
+        --infile /data/qct_2023-10-20_19h40.38.964_2_session_1.EDF \
+        --bidsprefix /output/session01/func/sub-001_ses-001_task-qct_dir-LR \
+        -m /data/metadata/info_ET.json
+    ```
+- [ ] Copy all _eyetrack.tsv.gz and .json generated into your copy of the *DataLad* dataset in BIDS.
```
Comment (oesteban) [inline docs/data-management/post-session.md]: Just a nit pick
```suggestion
- [ ] Copy all `<prefix>_eyetrack.tsv.gz` and `<prefix>_eyetrack.json` generated into your copy of the *DataLad* dataset in BIDS.
```

**Review by oesteban — APPROVED** [review]
